licenses-binary/LICENSE-bouncycastle.html

```diff
@@ -0,0 +1,22 @@
+<html>
```

Comment:
it's a MIT license, copied from https://github.com/bcgit/bc-java/blob/main/LICENSE.html